src/main/java/hvac/roomcoordinator/behaviours/RoomCoordinatorTickerBehaviour.java

```diff
@@ -22,33 +24,34 @@ public RoomCoordinatorTickerBehaviour(Agent a, long period, RoomContext roomCont
 
     @Override
     protected void onTick() {
+        LocalDateTime now = DateTimeSimulator.getCurrentDate();
```

Comment:
I think it will be easier to do it the way it is done on master

Reply:
Sure